src/panels/lovelace/badges/hui-summary-badge.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  customElement,
+  html,
+  LitElement,
+  TemplateResult,
+  property,
+} from "lit-element";
+import { computeDomain } from "../../../common/entity/compute_domain";
+import "../../../components/ha-label-badge";
+import { HomeAssistant } from "../../../types";
+import { LovelaceBadge } from "../types";
+import { SummaryBadgeConfig } from "./types";
+
+@customElement("hui-summary-badge")
+export class HuiSummaryBadge extends LitElement implements LovelaceBadge {
+  @property({ attribute: false }) public hass?: HomeAssistant;
+
+  @property() protected _config?: SummaryBadgeConfig;
+
+  public setConfig(config: SummaryBadgeConfig): void {
+    this._config = {
+      icon: "hass:lightbulb-group",
+      domain: "light",
```

Comment:
We should not default this to the light domain I think if it is a generic summary badge?